src/coreclr/vm/stubgen.cpp

```diff
@@ -2669,7 +2669,7 @@ void ILStubLinker::TransformArgForJIT(LocalDesc *pLoc)
             // JIT will handle structures
             if (pLoc->InternalToken.IsValueType())
             {
-                _ASSERTE(pLoc->InternalToken.IsBlittable());
+                _ASSERTE(pLoc->InternalToken.IsNativeValueType() || !pLoc->InternalToken.GetMethodTable()->ContainsPointers());
```

Comment:
Should this predicate be put into a helper called `IsUnmanaged()`?

Reply:
We could add an `IsUnmanaged` method to `TypeHandle` that checks these two cases, though it would only be used here so I'm not sure how useful it would be. I'll consider it.